cmd/maya-apiserver/app/server/http.go

```diff
@@ -120,6 +120,17 @@ var (
 		[]string{"code", "method"},
 	)
 
+	latestOpenEBSPoolRequestDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "latest_openebs_pool_request_duration_seconds",
+			Help:    "Request response time of the /latest/pool/.",
+			Buckets: []float64{0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.5, 1, 2.5, 5, 10},
```

Comment:
I would prefer some comments here for better understanding. It helps new comers.

Reply:
done